README.rst

```diff
@@ -1,7 +1,10 @@
 supernova
 =========
 
-  Tool for exporting GitHub stars as an organized list.
+.. image:: https://travis-ci.org/0xazure/supernova.svg?branch=master
+    :target: https://travis-ci.org/0xazure/supernova
+
+A tool for exporting GitHub stars as an organized list.
```

Comment:
```suggestion
    Tool for exporting GitHub stars as an organized list.
```

Minor nit with the indentation here, it needs to be indented with 4 spaces to be rendered as a `blockquote`.

Reply:
Note the indentation doesn't let the badge render: https://github.com/0xazure/supernova/blob/114bb0cc15e94b9c8c99ac189c7e02df40e040b6/README.rst

I'm not experienced enough with `rst` to understand why it's not there but with #10 opened, it could be a `good-first-bug` to slap in the `.md` version of the badge: `[![Build Status](https://travis-ci.org/0xazure/supernova.svg?branch=master)](https://travis-ci.org/0xazure/supernova)`

Reply:
Oh wild, yeah it doesn't render anything.

I ran the text through an offline parser and got this message:

```text
System Message: ERROR/3 (supernova/README.rst, line 4)

Error in "image" directive: no content permitted.

.. image:: https://travis-ci.org/0xazure/supernova.svg?branch=master
    :target: https://travis-ci.org/0xazure/supernova

    Tool for exporting GitHub stars as an organized list.
```

Because `:target:` and `Tool for [...]` are at the same indent level, I think it's trying to parse everything as one block instead of looking at it as separate directives.

Adding [a `|` character](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#line-blocks) in between the image and text should force it to treat them as separate blocks, at least as a temporary fix. #10 will definitely be the complete fix in the end.